sys-firmware/ipxe/ipxe-1.21.1_p20230601.ebuild

```diff
@@ -96,8 +96,8 @@ src_compile() {
 		ipxemake bin/15ad07b0.rom # vmxnet3
 	fi
 
-	use efi && ipxemake PLATFORM=efi BIN=bin-i386-efi bin-i386-efi/ipxe.efi
-	use efi64 && ipxemake PLATFORM=efi BIN=bin-x86_64-efi bin-x86_64-efi/ipxe.efi
+	use uefi && ipxemake PLATFORM=efi BIN=bin-i386-efi bin-i386-efi/ipxe.efi
+	use uefi64 && ipxemake PLATFORM=efi BIN=bin-x86_64-efi bin-x86_64-efi/ipxe.efi
```

Comment:
No underscores in USE flags please. Use hyphens (-) instead.

Reply:
`sys-apps/memtest86+` uses `efi32` and `efi64`. Might as well use those to match.

Reply:
Or alternatively rename `sys-apps/memtest86+`'s flags to have the `u` in them. I have no preference other than keeping things consistent.

Reply:
I wonder if we need USE `uefi, `uefi32`, and `uefi64`.  Because I think that for xen, the UEFI USE flag should be neither `uefi32` nor `uefi64`.

Reply:
I wonder whether `uefi-i386` makes a lot of sense on `amd64`? I know of some oddball Intel boards that had 32-bit UEFI implementations but 64-bit CPUs, but those were pretty obscure

Reply:
> I wonder whether `uefi-i386` makes a lot of sense on `amd64`?

I think it may does, at least in the case of ipxe. If I understand the situation correctly then USE=uefi-i368 will install the IPXE which allows booting whatever system requires a i386 UEFI IPXE binary to netboot.

Reply:
Alternatively we can also just have one flag (`uefi`) toggle both. But I don't know what the build time of this package is so maybe this is a stupid idea.

Reply:
Another alternative would be to use the [UEFI image type](https://uefi.org/specs/UEFI/2.10/03_Boot_Manager.html#uefi-image-types) as a basis for naming the USE flags.

amd64 = uefi-x64
arm = uefi-arm
arm64 = uefi-aa64
ia64 = uefl-ia64
riscv32 = uefi-riscv32
riscv64 = uefi-riscv64
x86 = uefi-ia32

Reply:
> Alternatively we can also just have one flag (`uefi`) toggle both. But I don't know what the build time of this package is so maybe this is a stupid idea.

I checked and the build time for both this package and `sys-apps/memtest86+` is rather low, so I think we can get away with keeping things simple and just have the one `uefi` flag for both 32 and 64 bit versions.

Reply:
and uefi-ia32 on amd64 doesn't require any multilib support in gcc?

Reply:
This I don't know. My guess would be that it does.